### ISSUE TITLE
Ai opponents are really easy to beat

### ISSUE THREAD

**Issue by Sabersabersaber123** [shishkabob27/CardWars#33]
unsure if the ai was like this in the original, but i think it needs to be improved as sometimes it just replaces existing cards and other dumb stuff

**Comment by bpdrej** [comment]
> unsure if the ai was like this in the original, but i think it needs to be improved as sometimes it just replaces existing cards and other dumb stuff

Yes it's always been pretty dumb. Though maybe with this recompiled version some things were messed up
